assessment/asr_validation/wald2017b/analysis/A1-biaxial.i

```diff
@@ -1005,12 +1005,12 @@
   petsc_options_iname = '-pc_type'
   petsc_options_value = 'lu'
   start_time = 2419200
-  dt = 1000000
+  dt = 500000
```

Comment:
I'm a little concerned about cutting the timestep size if we don't really need to just because it will double the run time. Were you seeing significant time integration error? Or is it because it helps the model converge better to those tighter tolerances? If there's a strong case for doing that, I'm fine with it, but I just wanted to check first.

Reply:
Yeah, I had to do this to make all the cases converge consistently with a tighter tolerance. Even with the loose tolerances, the timestep used earlier was causing convergence issues which lead to its reduction anyway. It also was responsible for the assessment cases to diff during nightly runs.

Reply:
With the reduction in timestep, the computational time increases. But we are still within the maximum limit we allowed for these cases.

Reply:
OK, thanks. Let's go with the smaller time step then. I'll merge this.